### PR TITLE
Adds docs for skipIndexes option

### PR DESCRIPTION
**Pull request by egalpin**
Adds query option docs for `skipIndexes`. Relates to https://github.com/apache/pinot/pull/12414

cc @npawar

### CODE REVIEW

File: users/user-guide-query/query-options.md [hunk 1]
```diff
@@ -57,6 +57,16 @@ description: This document contains all the available query options
         See <a data-mention href="grouping-algorithm.md#configuration-parameters">#configuration-parameters</a></td>
       <td>Server level config</td>
     </tr>
+    <tr>
+       <td><strong>skipIndexes</strong></td>
+       <td>
+         <p>Which indexes to skip usage of (i.e. scan instead), per-column.  This is useful for side-by-side comparison/debugging.  There can be cases where the use of an index is actually more expensive than performing a scan of the docs which match other filters.  One such example could be a low-selectivity inverted index used in conjunction with another highly selective filter.</p>
```
Comment (Jackie-Jiang) [inline users/user-guide-query/query-options.md]: (minor) Should we use double spaces after `.`?

File: users/user-guide-query/query-options.md [hunk 2]
```diff
@@ -57,6 +57,16 @@ description: This document contains all the available query options
...
+         <p>Config can be specified using url parameter format: <code>skipIndexes='col1=inverted,range&col2=inverted'</code></p>
```
Comment (Jackie-Jiang) [inline users/user-guide-query/query-options.md:64]: It would be helpful if we can list down all indexes that can be skipped

**Review by Jackie-Jiang — COMMENTED** [review]
Thank you!

File: users/user-guide-query/query-options.md (hunk 2 above)
Comment (egalpin) [reply users/user-guide-query/query-options.md:64]: Agreed!  Added in https://github.com/pinot-contrib/pinot-docs/pull/340/commits/88f30535973f86a453e5cb0e879d2e2e53e1fe00

**Pull request merged**